assets/scss/components/_button.scss

```diff
@@ -1,47 +1,144 @@
-/* ==========================================================================
-   #BUTTONS
-   ========================================================================== */
+// /* ==========================================================================
+//    #BUTTONS
+//    ========================================================================== */
+
+@mixin button_transition() {
+    transition: all 0.2s ease-in-out;
+    &:hover,
+    &:focus {
+        transition: all 0.2s ease-in-out;
+    }
+}
+
+@mixin button_focus($color, $shadow-color) {
+    &:focus {
+        background: $color;
+        box-shadow: 0px 0px 0px 4px $shadow-color;
+        color: $color-white;
+    }
+}
+
+@mixin button_disabled() {
+    &:disabled {
+        opacity: 0.48;
+    }
+}
+
+@mixin button_tertiary_base() {
+    padding: 12px 0;
+    background: $color-white;
+    text-align: center;
+    font-family: $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+
+    @include button_transition();
+}
 
 .c-button {
-    background-color: transparent;
-    border-color: $color-grey-450;
-    border-radius: 4px;
-    border-style: solid;
-    border-width: rem-calc(1);
-    color: $color-medium-grey;
-    cursor: pointer;
-    display: inline-block;
-    font-family: Montserrat, Arial, Helvetica, sans-serif;
-    font-size: 1rem;
-    font-weight: 400;
-    line-height: normal;
-    margin: 0 0 1rem;
-    outline: none;
-    padding: 0.85714rem 2.28571rem;
-    position: relative;
+    color: $color-white;
     text-align: center;
+    font-family: Assistant, $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+    outline: 0;
+    border: 0;
+    padding: 12px 16px;
+    cursor: pointer;
```

Comment:
Consider using a unified order of properties. It could be alphabetical or grouped by type, that way the code will look more organized.